web/styles/recent_view.css

```diff
@@ -95,7 +93,32 @@
             expensive repaint. The downside of not doing so is that the scrollbar is not visible to user when
             user is at the bottom of scroll container when the compose box is open.
             */
-            margin-bottom: var(--max-unexpanded-compose-height);
+            margin-bottom: calc(var(--max-unexpanded-compose-height) - 71px);
+            /* TODO(evy) find a cleaner way to account for height of the banner + its margin */
```

Comment:
This got messy because the banner has a height of 41px or something, and that's determined through styling outside the scope of this PR. We also don't need to be exact here, since we're just trying to leave space for the compose box.